### PR TITLE
DT-1107: added annotations for alerts and notifications for DSP tech services

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/case-notes-to-probation-dev/00-namespace.yaml [hunk 1]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "case-notes-to-probation"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/case-notes-poll-push.git"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/case-notes-to-probation-dev/00-namespace.yaml:12]: should be https://github.com/ministryofjustice/case-notes-to-probation

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/case-notes-to-probation-prod/00-namespace.yaml [hunk 2]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "case-notes-to-probation"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/case-notes-poll-push.git"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "dps_alerts_non_prod"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/case-notes-to-probation-prod/00-namespace.yaml:13]: should be `dps_alerts`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-case-notes-dev/00-namespace.yaml [hunk 3]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "offender-case-notes"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-case-notes.git"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "dps_alerts"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-case-notes-dev/00-namespace.yaml:13]: should be `dps_alerts_non_prod`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml [hunk 4]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "offender-events"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml:12]: should be https://github.com/ministryofjustice/prison-offender-events and https://github.com/ministryofjustice/probation-offender-events, with the `dev` version also including https://github.com/ministryofjustice/offender-events-ui

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-api-certificates/00-namespace.yaml [hunk 5]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "digital-prison-services"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/prisonstaffhub.git"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-api-certificates/00-namespace.yaml:12]: guess this should link to auth and also to prison api, either that or leave blank :thinking:

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-to-nhs-update-prod/00-namespace.yaml [hunk 6]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "prison-to-nhs-update"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/prison-to-nhs-update.git"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "dps_alerts_non_prod"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-to-nhs-update-prod/00-namespace.yaml:13]: `dps_alerts`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-offender-search-dev/00-namespace.yaml [hunk 7]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "probation-offender-search"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-search.git"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-offender-search-dev/00-namespace.yaml:12]: should be probation-offender-search and also for dev include probation-offender-search-indexer

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-offender-search-prod/00-namespace.yaml [hunk 8]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "probation-offender-search"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-search.git"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "dps_alerts_non_prod"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-offender-search-prod/00-namespace.yaml]: `dps_alerts`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/token-verification-api-prod/00-namespace.yaml [hunk 9]
```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "token-verification-api"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/token-verification-api.git"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "dps_alerts_non_prod"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/token-verification-api-prod/00-namespace.yaml]: `dps_alerts`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml [hunk 10]
```diff
@@ -9,4 +9,7 @@ metadata:
     cloud-platform.justice.gov.uk/business-unit: "HMPPS"
     cloud-platform.justice.gov.uk/application: "offender-events"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
-    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git, https://github.com/ministryofjustice/probation-offender-events.git, https://github.com/ministryofjustice/offender-events-ui.git"
```
Comment (petergphillips) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml:12]: almost, https://github.com/ministryofjustice/offender-events should be https://github.com/ministryofjustice/prison-offender-events

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by imran-awan1 — APPROVED** [review]

**Pull request merged**